### DISCUSSION TITLE
Planetscale CLI - handshake error

### DISCUSSION THREAD

**Discussion by SrinivasMajeti** [planetscale/discussion#171] [Q&A] [answered]
Hello, I am exploring planetscale and create a DB in UI and trying to connect to the DB from CLI and getting the below error

**_pscale shell majetitest main_**
**2022-02-23T14:16:01-05:00	error	error proxying conns	{"error": "couldn't initiate TLS handshake to remote addr: read tcp 172.20.235.58:54731->35.168.234.113:3307: read: connection reset by peer"}**

Followed the document https://docs.planetscale.com/concepts/planetscale-environment-setup

Please help.

**Comment by jgreet** [comment, marked as answer]
Is there any chance you installed an older version of the CLI? Current is `0.89.0` and you can check with `pscale --version`. 

If you create a [username and password](https://docs.planetscale.com/concepts/connection-strings) are you able to connect using the `mysql` CLI client? 

From the text of the error it looks like either an SSL issue, in which case I would expect to hear reports from multiple which hasn't happened yet, or a network issue between your computer and our servers. Perhaps port 3307 is blocked on your network?

**Comment by SrinivasMajeti** [comment]
Thank you. looks like the later. Port is blocked from my office network. Works like a charm from home PC.
